README.md

```diff
@@ -177,6 +205,8 @@ $ npx cypress run --env grep="-hello world"
 $ npx cypress run --env grep="hello; -world"
 ```
 
+Note: An inverted name filter that matches a suite name does not exclude its tests.
```

Comment:
This wasn't in the docs before, and it's nice to know that this doesn't (currently) work.